dtests/src/test/scala/io/snappydata/hydra/northwind/NWTestUtil.scala

```diff
@@ -103,6 +68,22 @@ object NWTestUtil {
     Row.fromSeq(md)
   }
 
+  def writeToFile(df: DataFrame, dest: String, snc: SnappyContext): Unit = {
+    import snc.implicits._
+    df.map(dataTypeConverter)(RowEncoder(df.schema))
+      .map(row => {
+        val sb = new StringBuilder
+        row.toSeq.foreach(e => {
+          if (e != null)
+            sb.append(e.toString).append(",")
+          else
+            sb.append("NULL").append(",")
+        })
+        sb.toString()
+      }).write.format("org.apache.spark.sql.execution.datasources.csv.CSVFileFormat").option(
```

Comment:
Yes, using CSVFileFormat makes it very slow which results in increased test time and that's the reason, Sumedh has changed this code in dunit. I will incorporate the same in northwind later on in other PR.